### ISSUE TITLE
Pi.Alert website doesn't work

### ISSUE THREAD

**Issue by antoinerincent** [pucherot/Pi.Alert#20]
Hey! Followed the installation steps (except pihole, which I have running in a docker container for only certain devices), and everything worked perfectly. However when I try to reach http://pi.alert/ I get nothing. If I try to log into the pi running it all directly, I get a 403 Forbidden error, how can I fix this issue? I'm guessing there is a specific port that I should provide to connect directly to it with the IP adress, what would that be?

Thanks again for the great work, the service looks super cool, great job! :)

**Comment by will6789** [comment]
Hi, Heres a couple of suggestions:
Are you able to reach it on `<IP Address>/pialert`?
The http://pi.alert address relies on your DNS server being configured to reply with the correct IP address, have you configured your pihole with the pi.alert DNS record and is your device using the pihole as its DNS server?

**Comment by antoinerincent** [comment]
Trying `<IP Address>/pialert` also results in a 403 Forbidden
I also tried adding a DNS rule in my Pihole instance for the pi.alert to redirect to the IP, but it gives me the error that the URL isn't a valid one, it's more of a pihole issue than a Pi.Alert issue though, I'll keep checking it out!
Thanks!

**Comment by jasonwtobe** [comment]
Same issue here until I saw this thread and used `<IP Address>/pialert`, where then it worked. May be useful to add this to the `INSTALL.md` to reduce confusion for those of use who aren't very technical.

**Comment by pucherot** [comment]
As will6789 says you should configure the DNS record in pi.hole

This is written in the installation part of pi.hole (maybe you skipped it because you already had pi.hole installed)

Install.md updated with a clarification.

**Comment by mrchoc** [comment]
getting similar problem, where ### http://pi.alert gives me a 403 error, while `<IP Address>/pialert` sends me to the "Did you mean to go to the admin panel?" page from the pi hole

**Comment by pucherot** [comment]
please test http://192.168.1.x/pialert
and test ping pi,alert

**Comment by mrchoc** [comment]
> getting similar problem, where http://pi.alert gives me a 403 error, while `<IP Address>/pialert` sends me to the "Did you mean to go to the admin panel?" page from the pi hole

Sorry, what do you mean by test http://192.168.1.x/pialert? I already tried that and I got the "Did you mean to go to the admin panel?" page.

And pinging pi.alert doesnt gives could not find host error

**Comment by pucherot** [comment]
Hi mrchoc

If ping to pi.alert is not working is becouse your DNS can't resolve the name. The problem may be:
1.  Your machine are using a different DNS (no Pi.hole DNS server). please, check your IP configuration with (ifconfig, ipconfig /all, nslookup pi.alert)
2. You forgot add the DNS Local record to your Pi.hole Configuration:
   - https://github.com/pucherot/Pi.Alert/blob/main/docs/INSTALL.md#pi-hole-setup
   - Point 7

In the other hand, can you paste a screeshot of your browser with the error accessing http://192.168.1.x/pialert?

Thanks

**Comment by mrchoc** [comment]
![image](https://user-images.githubusercontent.com/65016628/104738061-010d2080-5780-11eb-9fd8-3c40136481e1.png)
This is page I get when accessing http://192.168.1.x/pialert 

I tried pinging pi.alert and it worked this time, but when I try accessing pi.alert in the browser I get a 403 error

**Comment by mrchoc** [comment]
okay, I have found the solution.

In step 2 of the Pi.alert setup, `sudo ln -s /home/pi/pialert/front /var/www/html/pialert` creates a front folder inside the /var/www/html/pialert directory.

Since the external.conf accesses the `/var/www/html/pialert` as the root directory, all it sees is the front folder, which might be causing the 403 forbidden error.

tldr, the solution is to 
1. `cd /etc/lighttpd`
2. `sudo nano external.conf`
3. change `server.document-root = "/var/www/html/pialert"` to `server.document-root = "/var/www/html/pialert/front"`
4. `systemctl restart lighttpd.service`
5. restart pi.alert and it should work

**Comment by pucherot** [comment]
I think that the instructions in the actual installation guide solves this issue.

**Comment by N8Deathrider** [comment]
> okay, I have found the solution.
> 
> In step 2 of the Pi.alert setup, `sudo ln -s /home/pi/pialert/front /var/www/html/pialert` creates a front folder inside the /var/www/html/pialert directory.
> 
> Since the external.conf accesses the `/var/www/html/pialert` as the root directory, all it sees is the front folder, which might be causing the 403 forbidden error.
> 
> tldr, the solution is to
> 
> 1. `cd /etc/lighttpd`
> 2. `sudo nano external.conf`
> 3. change `server.document-root = "/var/www/html/pialert"` to `server.document-root = "/var/www/html/pialert/front"`
> 4. `systemctl restart lighttpd.service`
> 5. refresh pi.alert and it should work

@mrchoc I am experiencing the same issue you are, I followed the installation instructions to a T and am still running into the `403 Forbidden` error when accessing `https://pi.alert` and I'm getting the `Pi-hole: Your black hole for Internet advertisements
Did you mean to go to the admin panel?` page when trying to access it from `http://192.168.1.x/pialert`. I attempted to follow your solution but editing the file `external.conf` (which is empty for me and because I'm unfamiliar with Lighttpd I don't know if that's normal)  just caused Lighttpd to error out and fail to start. I undid that change and recognized the line `server.document-root = "/var/www/html/pialert"` being inside `/etc/lighttpd/conf-available/pialert_front.conf` and made the change there which did allow Lighttpd to restart but did not fix anything. Are your solution instructions all you did?

 @pucherot Currently, I've reverted my modifications back to step 5.11 of the installation instructions. I see you think that the instructions in the installation guide solve the issue, if so what do you believe I did incorrectly?
